### DISCUSSION TITLE
How to create the ending post section?

### DISCUSSION THREAD

**Discussion by hellt** [uPagge/uBlogger#81] [Q&A] [answered]
### Describe the feature you want

Hi @uPagge 
Could you please share with us how to create the section after the post, you have it on uPagge.ru

![image](https://user-images.githubusercontent.com/5679861/115233949-ce751a00-a118-11eb-9cb8-b308cb7cba93.png)

**Comment by uPagge** [comment]
It is better to write such questions in discussions. I transfer there

**Comment by uPagge** [comment, marked as answer]
To make a separate block under the post, I left a special partial. In the template itself, it is empty. It is required to override

![изображение](https://user-images.githubusercontent.com/40397740/115255953-7699db80-a137-11eb-954e-2446f40c84b5.png)

![изображение](https://user-images.githubusercontent.com/40397740/115255989-80234380-a137-11eb-84b8-167eccf1b947.png)

```
<aside class="article-post">
    <div class="content-block content-block-position block-donate">
        <div class="post">
```

This allows any post-post block to be rendered with a CSS override and this partial

**Reply by hellt** [reply]
Thanks @uPagge 
Got it running!
